File: src/Python/easy/array/problem2540.py
from typing import List

class Solution:
    def getCommon(self, nums1: List[int], nums2: List[int]) -> int:
        nums_set = set(nums2)
        print(nums_set)
        for num in nums1:
            if num in nums_set:
                return num
        return -1


nums1 = [1,2,3,6]; nums2 = [2,3,4,5]

solution = Solution()
solution.getCommon(nums1, nums2)

File: src/JavaScript/easy/array/problem2540.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
var getCommon = function (nums1, nums2) {
    let set = new Set(nums2)
    for (let i = 0; i < nums1.length; i++) {
        if (set.has(nums1[i])) {
            return nums1[i]
        }
    }
    return -1
};

// let nums1 = [1,2,3], nums2 = [2,4]
let nums1 = [1,2,3,6], nums2 = [2,3,4,5]

getCommon(nums1, nums2)